kafka-rest/src/main/java/io/confluent/kafkarest/resources/v3/ProducerMetrics.java

```diff
@@ -50,339 +40,226 @@
-  }
+  // response
+  static final String RESPONSE_SEND_RATE_METRIC_NAME = "response-rate";
+  private static final String RESPONSE_SEND_RATE_METRIC_DOC =
+      "The average number of responses sent per second.";
 
-  private static final class BeanCoordinate {
-    final String beanName;
-    final Map<String, String> tags;
+  static final String RESPONSE_COUNT_WINDOWED_METRIC_NAME = "response-count-windowed";
+  private static final String RESPONSE_COUNT_WINDOWED_METRIC_DOC =
+      "The total number of responses sent in the given window.";
 
-    BeanCoordinate(String beanName, Map<String, String> tags) {
-      this.beanName = requireNonNull(beanName);
-      this.tags = ImmutableMap.copyOf(tags);
-    }
+  // errors
+  static final String ERROR_COUNT_WINDOWED_METRIC_NAME = "error-count-windowed";
+  private static final String ERROR_COUNT_WINDOWED_METRIC_DOC =
+      "The total number of record sends that resulted in errors in the given window.";
 
-    @Override
-    public boolean equals(Object o) {
-      if (this == o) {
-        return true;
-      }
-      if (!(o instanceof BeanCoordinate)) {
-        return false;
-      }
-      BeanCoordinate that = (BeanCoordinate) o;
-      return Objects.equals(beanName, that.beanName) && Objects.equals(tags, that.tags);
-    }
+  static final String RECORD_ERROR_RATE_METRIC_NAME = "record-error-rate";
+  private static final String RECORD_ERROR_RATE_METRIC_DOC =
+      "The average per-second number of record sends that resulted in errors.";
 
-    @Override
-    public int hashCode() {
-      return Objects.hash(beanName, tags);
-    }
-  }
+  // latency
+  static final String REQUEST_LATENCY_AVG_METRIC_NAME = "request-latency-avg";
+  private static final String REQUEST_LATENCY_AVG_METRIC_DOC = "The average request latency";
 
-  /**
-   * A class representing a JMX MBean where each metric maps to an MBean attribute. Sensors should
-   * be added via the {@code sensor} methods on this class, rather than directly through the {@link
-   * Metrics} class, so that the sensor names are made to be unique (based on the MBean name) and so
-   * the sensors are removed when this group is {@link #close() closed}.
-   */
-  final class ProduceMetricMBean implements AutoCloseable {
-    private final BeanCoordinate beanCoordinate;
-
-    /**
-     * Create a produce metrics MBean.
-     *
-     * @param beanCoordinate the identifier of the bean; may not be null and must be valid
-     */
-    ProduceMetricMBean(BeanCoordinate beanCoordinate) {
-      this.beanCoordinate = requireNonNull(beanCoordinate);
-    }
+  static final String REQUEST_LATENCY_MAX_METRIC_NAME = "request-latency-max";
+  private static final String REQUEST_LATENCY_MAX_METRIC_DOC = "The max request latency";
 
-    void recordResponse() {
-      recordMetric(fullyQualifiedResponseSensor, 1.0);
-    }
+  static final String REQUEST_LATENCY_PCT_METRIC_PREFIX = "request-latency-avg-";
```

Comment:
nit: Remove `-avg`, percentiles and averages don't make too much sense together.

Reply:
Done. Note that the resulting JMX metric name will be different. So, if we are using that in any dashboard, it will need to be updated.